data/coalition jobs.txt

```diff
@@ -1105,7 +1105,7 @@ mission "Coalition: From Games 1"
 		"coalition jobs" ++
 		payment
 		payment 10000
-		dialog `The passengers thank you for bringing them to <planet> and pay you <payment>.`
+		dialog `The passengers thank you for bringing them to <planet> and pay you <payment>. From their overall manner, it looks like the Coalition Games were quite entertaining this year.`
```

Comment:
"Based on their behaviour, [you think/it seems/it looks like] the Coalition Games were quite entertaining."
basically rephrased the start, the part after the , is just other options, but notice how I ended the sentence without "this year"
did the game tell us the Coalition Games were yearly? if not, maybe not put it here